### PR TITLE
WIP to show modifying the ZK Timeout via System property.

### PR DESCRIPTION
**Pull request by jnh5y**

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala [hunk 1]
```diff
@@ -151,7 +152,17 @@ object AccumuloDataStoreFactory {
     if (useMock) {
       new MockInstance(instance).getConnector(user, authToken)
     } else {
-      new ZooKeeperInstance(instance, zookeepers).getConnector(user, authToken)
+      // NB: For those wanting to set this via JAVA_OPTS, this key is "instance.zookeeper.timeout" in Accumulo 1.6.x.
+      val clientConfiguration = if (System.getProperty(ClientProperty.INSTANCE_ZK_TIMEOUT.getKey) != null) {
+        new ClientConfiguration()
+          .withInstance(instance)
+          .withZkHosts(zookeepers)
+          .`with`(ClientProperty.INSTANCE_ZK_TIMEOUT, System.getProperty(ClientProperty.INSTANCE_ZK_TIMEOUT.getKey))
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala]: do we need the ticks?

Comment (jnh5y) [inline geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala]: Ayup, 'with' is reserved keyword in Scala.

**Comment by jahhulbert-ccri** [comment]
looks good

**Comment by elahrvivaz** [comment]
closing in favor of #912

**Pull request closed without merging**